funcx_endpoint/funcx_endpoint/executors/high_throughput/executor.py

```diff
@@ -496,17 +497,24 @@ def _start_local_interchange_process(self):
             },
         )
         self.queue_proc.start()
+        msg = None
         try:
-            (self.worker_task_port, self.worker_result_port) = comm_q.get(
-                block=True, timeout=120
-            )
-            comm_q.close()  # not strictly necessary, but be plain about intentions
-            comm_q.join_thread()
-            del comm_q
+            msg = comm_q.get(block=True, timeout=120)
         except queue.Empty:
-            log.error("Interchange has not completed initialization in 120s. Aborting")
+            log.error("Interchange did not complete initialization.")
+
+        if not msg:
+            # poor-person's attempt to not interweave the traceback log lines
+            # with the subprocess' likely traceback lines
+            time.sleep(0.5)
```

Comment:
I would have done 0.1 but maybe it's not enough?

Reply:
It's a good thought (I thought about same);  0.1 is probably just fine, but I'm also not particularly worried about it &rarr; we're shutting down (dying!) anyway.  I'd rather err on the side of "potentially overloaded, so give more time."  If you feel strongly about it, we can follow up in a later PR, but I'm inclined to leave it at 0.5s for now.